features/routing/f5-router.feature

```diff
@@ -0,0 +1,33 @@
+Feature: F5 router related scenarios
+
+  # @author hongli@redhat.com
+  # @case_id OCP-24080
+  @admin
+  @destructive
+  Scenario: the f5 router image should contains openssh-clients package
+    Given I switch to cluster admin pseudo user
+    And I use the router project
+    And default router image is stored into the :router_image clipboard
+    And default router replica count is restored after scenario
+    When I run the :scale client command with:
+      | resource | dc     |
+      | name     | router |
+      | replicas | 0      |
```

Comment:
This is going to work only with `dc` router. I thought that we have a special step. Are we ever using non-dc routers? If so it is worth writing a step. Maybe after PR is merged.

Reply:
We are always using dc for routers (haproxy and f5) in 3.x, but using deployment since 4.1. If needed we can have a step and update related scenarios later.

Reply:
Got it. we can keep as is for the time being.